app.html

```diff
@@ -675,9 +675,12 @@
 }
 
 async function AddPreset() {
+  var r = confirm("Duplicate current preset?");
+  if (r == true) {
   Send("duplicate_preset " + presets.length);
```

Comment:
This line (and the following two) now needs to be indented two steps

Reply:
done